server/src/client/app/src/pages/docs/About.js

```diff
@@ -86,7 +86,7 @@ const BigBadge = styled(Badge)`
 `;
 
 const tc_ids = [1, 2, 27, 86, 348, 970];
-const core_ids = [1, 2, 27, 86, 348, 970, 1140, 869, 8111, 9186, 3744];
+const core_ids = [1, 2, 27, 86, 348, 970, 1140, 869, 8111, 9186, 3744, 35875];
```

Comment:
```suggestion
const core_ids = [1, 2, 27, 86, 348, 970, 1140, 869, 8111, 9186, 3744, 35875, 34097];
```
Also added Lennart, waiting for @josvandervelde and @Taniya-Das